zokrates_core/src/imports.rs

```diff
@@ -202,7 +203,14 @@ impl Importer {
     				},
     				s => return Err(CompileError::ImportError(Error::new(format!("Packing helper {} not found", s))))
     			}
-    		} else {
+			} /*else if import.source.starts_with("WASM/") {
```

Comment:
Just to clarify: the imports in ZoKrates are currently for ZoKrates functions. We don't have inline solvers (yet? #135), maybe we could use the imports to import those once we do...

This work is already useful without exposing it to the user (ie for ZoKrates-generated code) so this part is not crucial at this point IMHO.

Reply:
to remove